tests/scripts/basic-build-test.sh

```diff
@@ -121,13 +121,13 @@ echo
 echo '################ compat.sh ################'
 {
```

Comment:
CI job on `basic-build-test`, since the PR job doesn't do it: https://jenkins-mbedtls.oss.arm.com/job/mbedtls-release-new/789/